holoviews/plotting/bokeh/raster.py

```diff
@@ -156,10 +167,404 @@ def get_data(self, element, ranges, style):
                 sanitized = dimension_sanitizer(vdim.name)
                 data[sanitized] = ['-' if is_nan(v) else vdim.pprint_value(v)
                                    for v in aggregate.dimension_values(vdim)]
+
+        # Filter radial heatmap options
+        style = {k: v for k, v in style.items() if not
+                 any(g in k for g in RadialHeatMapPlot._style_groups.values())}
         return (data, {'x': x, 'y': y, 'fill_color': {'field': 'zvalues', 'transform': cmapper},
                        'height': 1, 'width': 1}, style)
 
 
+class RadialHeatMapPlot(CompositeElementPlot, ColorbarPlot):
+
+    clipping_colors = param.Dict(default={'NaN': 'white'}, doc="""
+        Dictionary to specify colors for clipped values.
+        Allows setting color for NaN values and for values above and below
+        the min and max value. The min, max, or NaN color may specify
+        an RGB(A) color as a either (1) a color hex string of the form
+        #FFFFFF or #FFFFFFFF, (2) a length-3 or length-4 tuple specifying
+        values in the range 0-1, or (3) a named HTML color.""")
+
+    start_angle = param.Number(default=np.pi/2, doc="""
+        Define starting angle of the first annulus segment. By default, begins 
+        at 12 o'clock.""")
+
+    padding_inner = param.Number(default=0.1, bounds=(0, 0.5), doc="""
+        Define the radius fraction of inner, empty space.""")
+
+    padding_outer = param.Number(default=0.05, bounds=(0, 1), doc="""
+        Define the radius fraction of outer space including the labels.""")
+
```

Comment:
@philippjfr I think I would prefer different names for these as the use of ``padding`` can get confused with the more general idea of ``padding`` we want to introduce. Maybe ``ratio_inner`` or ``fraction_inner`` and the same for the ``outer`` parameter?

Reply:
Yeah I agree. Not sure which I'd prefer though.

Reply:
``ratio`` is shorter than ``fraction`` and I think more accurate.

Reply:
Looking back at it now, I think I much prefer ``fraction``.

Reply:
Decided to use ``radius_inner`` and ``radius_outer``.